back-end/db/seeds/seeds.go

```diff
@@ -0,0 +1,113 @@
+package seeds
+
+import (
+	"../schema"
+	"github.com/jinzhu/gorm"
+)
+
+func Seed(db *gorm.DB) {
+	user := createUser("bobbyrossy@ross.ross", "Admin", "Bob", "Ross")
+	db.Create(&user)
+
+	customer := createCustomer("customer@email.com", "Dan", "Harmon", "19026652353", "")
+	customer1 := createCustomer("customer1@email.com", "George", "Martin", "19026652776", "")
+	db.Create(&customer)
+	db.Create(&customer1)
+
+	customerOrder := createCustomerOrder(user, customer, "")
+	customerOrder1 := createCustomerOrder(user, customer1, "")
+	db.Create(&customerOrder)
+	db.Create(&customerOrder1)
+
+	vendor := createVendor("vendor@company.biz", "Biz Company", "15555555555", "")
+	db.Create(&vendor)
+
+	item := createItem(vendor, "Bruschetta, but it's actually just olives", "You heard me.", "Warehouse", 52599)
+	db.Create(&item)
+
+	orderItem := createOrderItem(customerOrder, item, 52599, 1, 0)
+	db.Create(&orderItem)
+
+	vendorOrder := createVendorOrder(user, vendor, "Need more fake Bruschetta.")
+	db.Create(&vendorOrder)
+
+	address := createAddress(customer, "Bob", "Ross", "1440 Painter Ave.", "PaintScape", "12235658747", "Home", true)
+	db.Create(&address)
+}
+
+func createUser(email, role, firstName, lastName string) schema.Users {
```

Comment:
Does `string` indicate that all of the arguments are strings? Or just `lastName`?

Reply:
Everything preceding it, until it hits another type, so yes, it does.  As an example
(number1, number2 int, word1, word2, word3 string) is a better example of how it works.